### DISCUSSION TITLE
ECL Code Formatter

### DISCUSSION THREAD

**Discussion by yomane-lnrs** [hpcc-systems/vscode-ecl#434] [Ideas]
I am working on POC for ECL Code Formatter. The concept is similar to Prettier extension in VS Code. Please share if anyone tried making similar thing before and tested so can pace up the development.

**Comment by GordonSmith** [comment]
Personally I would look at extending Prettier to support ECL.  They have a plugin architecture:  https://prettier.io/docs/en/plugins specifically for this.
Let me know if you make any progress?
